### DISCUSSION TITLE
Graphql files not counted in github repo language breakdown.

### DISCUSSION THREAD

**Discussion by timthedev07** [github-linguist/linguist#5520] [Classification] [answered]
## Describe the bug
There's quite few `.graphql` files in my repository, however, it's not showing up in the language statistics here:
![Screen Shot 2021-08-14 at 5 04 58 PM](https://user-images.githubusercontent.com/65774333/129450625-2204f84e-864d-426b-b002-b908b192b2aa.png)

Proof of .graphql files.
![Screen Shot 2021-08-14 at 5 05 41 PM](https://user-images.githubusercontent.com/65774333/129450646-96e4bf34-64ba-421a-903b-1724e7663bad.png)

### Expected behavior

I was expecting to see the percentage of graphql files be at least a little bit less than CSS files.

**Comment by Nixinova** [comment, marked as answer]
Thats because GraphQL is classified as a data language instead of a programming or markup language (which are displayed in the language bar). If you want GraphQL to show up in the bar add `*.graphql linguist-detectable` to `.gitattributes.

**Reply by timthedev07** [reply]
Thanks for the clarification! However, the percentage is still not what I've expected.

**Comment by lildude** [comment]
Converting to discussion as this is really a question of clarification and not a bug
